pyspecProcScripts/first_level/fl_dummy.py

```diff
@@ -0,0 +1,14 @@
+"""provides object fl_dummy that can take the place of a figure list, but doesn't do anything.
+
+`fl=fl_dummy` is a better default than `fl=None`, because it doesn't require any conditionals
```

Comment:
if we said fl =fl where fl is figlist_var() that wouldn't require conditionals either would it? having trouble understanding why we wouldn't just say fl= None

Reply:
This avoids `if fl is not None:as

Reply:
so is fl still a kwarg for functions?